src/ape_init/_cli.py

```diff
@@ -33,6 +33,43 @@ def cli(cli_ctx, github):
             else:
                 folder.mkdir(exist_ok=False)
 
+        body = """
+# User Specified Ignored
+
+
+# Ape stuff
+.build/
+.pytest_cache
+__pycache__
+.env
+.cache
```

Comment:
I don't think all of this is our stuff, but happy to include most of them

Reply:
Ya, maybe let's have 2 sections:

```
# Ape stuf
.build/
.cache/

# Python stuff
<everything else>
```